bin/commands/init/certificate/index.ts

```diff
@@ -0,0 +1,531 @@
+              password: CERT_PARMS.pkcs12_password,
+              alias: CERT_PARMS.pkcs12_name_lc
+            },
+            truststore: {
+              type: 'PKCS12',
+              file: `${CERT_PARMS.pkcs12_directory}/${CERT_PARMS.pkcs12_name}/${CERT_PARMS.pkcs12_name}.truststore.p12`,
+              password: CERT_PARMS.pkcs12_password
+            },
+            pem: {
+              key: `${CERT_PARMS.pkcs12_directory}/${CERT_PARMS.pkcs12_name}/${CERT_PARMS.pkcs12_name_lc}.key`,
+              certificate: `${CERT_PARMS.pkcs12_directory}/${CERT_PARMS.pkcs12_name}/${CERT_PARMS.pkcs12_name_lc}.cer`,
+              certificateAuthorities: CERT_PARMS.yaml_pem_cas
+            }
+          }
+        }
+      };
+      jsonlib.updateZoweYamlFromObj(std.getenv('ZWE_CLI_PARAMETER_CONFIG'), updateObj);
+      common.printLevel2Message(`Zowe configuration is updated successfully.`);
+    } else {
+      common.printLevel1Message(`Update certificate configuration to ${std.getenv('ZWE_CLI_PARAMETER_CONFIG')}`);
+      common.printMessage(`Please manually update to these values:`);
+      common.printMessage(``);
+      common.printMessage(`zowe:`);
+      common.printMessage(`  certificate:`);
+      common.printMessage(`    keystore:`);
+      common.printMessage(`      type: PKCS12`);
+      common.printMessage(`      file: "${CERT_PARMS.pkcs12_directory}/${CERT_PARMS.pkcs12_name}/${CERT_PARMS.pkcs12_name}.keystore.p12"`);
+      common.printMessage(`      password: "${CERT_PARMS.pkcs12_password}"`);
+      common.printMessage(`      alias: "${CERT_PARMS.pkcs12_name_lc}"`);
+      common.printMessage(`    truststore:`);
+      common.printMessage(`      type: PKCS12`);
+      common.printMessage(`      file: "${CERT_PARMS.pkcs12_directory}/${CERT_PARMS.pkcs12_name}/${CERT_PARMS.pkcs12_name}.truststore.p12"`);
+      common.printMessage(`      password: "${CERT_PARMS.pkcs12_password}"`);
+      common.printMessage(`    pem:`);
+      common.printMessage(`      key: "${CERT_PARMS.pkcs12_directory}/${CERT_PARMS.pkcs12_name}/${CERT_PARMS.pkcs12_name_lc}.key"`);
+      common.printMessage(`      certificate: "${CERT_PARMS.pkcs12_diryectory}/${CERT_PARMS.pkcs12_name}/${CERT_PARMS.pkcs12_name_lc}.cer"`);
+      common.printMessage(`      certificateAuthorities: "${CERT_PARMS.yaml_pem_cas}"`);
+      common.printMessage(``);
+      common.printLevel2Message(`Zowe configuration requires manual updates.`);
+    }
+
+  } else if (certType == "JCERACFKS") {
+    // FIXME: how do we check if keyring exists without permission on RDATALIB?
+    // should we clean up before creating new
+    if (std.getenv('ZWE_CLI_PARAMETER_ALLOW_OVERWRITE') == "true") {
+      // warning
+      common.printMessage(`Warning ZWEL0300W: Keyring "safkeyring:///${CERT_PARMS.keyring_owner}/${CERT_PARMS.keyring_name}" will be overwritten during configuration.`);
```

Comment:
Will print `safkeyring:///USERID...`, (same as in shell version). It is already a little bit confusing, that `safkeyring` is used with two or four slashes and now even with three 😄 
ZWEL0300W not documented in `zwe init certificate` section

Reply:
3 slashes is certainly a bug to fix.
`.errors` files document which errors exist in which command
https://github.com/zowe/zowe-install-packaging/blob/eefffa903fc0d2730ce43f7a89e61f4e2971c60e/bin/commands/init/certificate/.errors

This doesnt contain 300w, but should. If you make that edit, then the docs site has some automation that puts it into the documentation.